src/wallets/IdentityWallet.ts

```diff
@@ -480,3 +484,22 @@ export function calculateIdentityAddress(
       .slice(2 + 2 * 12)
   return toChecksumAddress(address)
 }
+
+/**
+ * Generates a random nonce to use for meta transactions.
+ * The nonce fits into the range of ]2^255, 2^256[.
+ * This is an alternative to the up counting nonce (]0, 2^255[) without the need
+ * to query a [[TLProvider]].
+ */
+export function getRandomNonce(): string {
+  const random = BigNumber.random()
+  const nonce = random.multipliedBy(RANDOM_NONCE_RANGE).plus(MIN_RANDOM_NONCE)
```

Comment:
You should make sure the nonce is an integer: `nonce.integerValue(BigNumber.ROUND_DOWN)`

Reply:
Should not matter how it is processed afterwards. Correct?

Reply:
I'll do since I'll remove the awkward string generation (see [this
discussion](https://github.com/trustlines-protocol/clientlib/pull/373#discussion_r421555230)).